janis_core/tool/test_classes.py

```diff
@@ -36,6 +36,7 @@ def __init__(
         file_diff_source: Optional[str] = None,
         array_index: Optional[int] = None,
         suffix_secondary_file: Optional[str] = None,
+        additional_paras: Optional[Dict] = {},
```

Comment:
Just so that it is easier to understand, can we change `additional_paras` to `preprocessor_params`.